pootle/apps/pootle_format/default.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+
+POOTLE_FORMATS = [
+    ("po",
+     dict(title='Gettext PO file',
+          extension="po",
+          template_extensions=["po", "pot"])),
```

Comment:
In Gettext land there isn't be a concept of `source_language`, just templates.  Though the POT files could be in a language other than English I've only rarely seen it done like that.  The msgid is simply a key so from a programming perspective its not important.  I'm almost sure the Gettext tools wouldn't care (haven't checked though).

There is a model of using PO as a monolingual file, though Ive never seen that done in the wild.